app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt

```diff
@@ -862,7 +858,6 @@ class ChatRoomFragment : Fragment(), ChatRoomView, EmojiKeyboardListener, EmojiR
             )
 
             subscribeComposeTextMessage()
-            getUnfinishedMessage()
```

Comment:
Will the user be able to see its unfinished message after this removed function?

Reply:
No, I should have point this out on the description. It's on the commit message only.
This feature is not working properly, so I thought it would be better to temporarily disable this feature until we get to fix it.